components/compass-runtime-agent/internal/kyma/secrets/requestparameters.go

```diff
@@ -0,0 +1,67 @@
+package appsecrets
```

Comment:
We could think of some common library as there is similar code in Application (the only difference is data type).

Reply:
Yes, in different PR